README.md

```diff
@@ -86,34 +98,40 @@ A roadmap that describes the direction of CRI-O can be found [here](/roadmap.md)
 | -------------------------- | --------------------------------------- |
 | [crio(8)](/docs/crio.8.md) | OCI Kubernetes Container Runtime daemon |
 
-Note that kpod and its container management and debugging commands have moved to a separate repository, located [here](https://github.com/containers/podman).
+Note that kpod and its container management and debugging commands have moved
+to a separate repository, located [here](https://github.com/containers/podman).
```

Comment:
This is the only reference that I see to kpod in this entire page.  Perhaps time to remove it, or replace it with a mention of Podman?